signup-service/src/main/java/io/mosip/signup/services/RegistrationService.java

```diff
@@ -251,21 +251,17 @@ public RegisterResponse register(RegisterRequest registerRequest, String transac
         transaction.setRegistrationStatus(RegistrationStatus.PENDING);
         cacheUtilService.setStatusCheckTransaction(transactionId, transaction);
 
-        notificationHelper.sendSMSNotificationAsync(registerRequest.getUserInfo().getPhone(), transaction.getLocale(),
-                        REGISTRATION_SMS_NOTIFICATION_TEMPLATE_KEY, null)
-                .thenAccept(notificationResponseRestResponseWrapper ->
-                    log.debug(notificationLogging, notificationResponseRestResponseWrapper)
-                );
+        notificationHelper.sendSMSNotification(registerRequest.getUserInfo().getPhone(), transaction.getLocale(),
```

Comment:
During registration, it should be ok to send the notification in a separate thread and not wait until the notification is sent out to complete and return the registration response.